data/schemas/notesSchema.js

```diff
@@ -0,0 +1,29 @@
+const yup = require('yup');
+
+//note_id, created_at timestamp, and updated_at timestamp not included in schema because they are generated automatically for each note
```

Comment:
Is this comment supposed to stay in the PR?

Reply:
I was hoping to leave it in the PR to eliminate confusion of why the note_id, created_at, and updated_at entries weren't being validated.